windows/deploy_vm/deploy_vm_from_ova.yml

```diff
@@ -49,6 +49,9 @@
 
 - name: "Get VM info"
   include_tasks: ../../common/vm_get_vm_info.yml
+
+#- name: "Get VM's primary network adapter MAC address"
```

Comment:
Is it needed for windows ova installation?

Reply:
These two lines were commented, which can be a reminder when more steps are added for Windows OVF testing in future.